File: app/src/main/java/com/example/loopdeck/MainActivity.java
package com.example.loopdeck;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.burhanrashid52.photoeditor.EditImageActivity;

public class MainActivity extends AppCompatActivity {
    Button btn , btn2 , btn3;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn = findViewById(R.id.btn_gallery);
        btn2 = findViewById(R.id.btn_image);
        btn3 = findViewById(R.id.btn_video);


        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, com.luminous.pick.MainActivity.class));
            }
        });

        btn2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, EditImageActivity.class));
            }
        });


        btn3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, com.obs.marveleditor.MainActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/example/loopdeck/MainActivity.kt
package com.example.loopdeck

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.example.loopdeck.ui.recents.RecentsActivity
import com.loopdeck.photoeditor.EditImageActivity
import com.luminous.pick.MainActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initViews()
    }

    private fun initViews() {

        btn_gallery.setOnClickListener {
            startActivity(Intent(this@MainActivity, MainActivity::class.java))
        }

        btn_image.setOnClickListener {
            startActivity(
                Intent(
                    this@MainActivity,
                    EditImageActivity::class.java
                )
            )
        }
        btn_video.setOnClickListener {
            startActivity(
                Intent(
                    this@MainActivity,
                    com.obs.marveleditor.MainActivity::class.java
                )
            )
        }

        btn_recents.setOnClickListener {
            startActivity(
                Intent(
                    this@MainActivity,
                    RecentsActivity::class.java
                )
            )
        }
    }
}